src/Ledger/Conway/Conformance/Certs.agda

```diff
@@ -175,7 +179,7 @@ data _⊢_⇀⦇_,CERTBASE⦈_ : CertEnv → CertState → ⊤ → CertState →
       ⟦ e , pp , vs , wdrls ⟧ᶜ ⊢
       ⟦ ⟦ voteDelegs , stakeDelegs , rewards , ddep ⟧ᵈ
       , stᵖ
-      , ⟦ dreps , ccHotKeys , gdep ⟧ᵛ
+      , ⟦ dReps , ccHotKeys , gdep ⟧ᵛ
       ⟧ᶜˢ
       ⇀⦇ _ ,CERTBASE⦈
       ⟦ ⟦ voteDelegs , stakeDelegs , constMap wdrlCreds 0 ∪ˡ rewards , ddep ⟧ᵈ
```

Comment:
The removal of delegations is still missing from here

Reply:
I forgot about the duplicate code. 🤦🏼

Will fix this today.